src/main/java/org/apache/sysds/runtime/compress/colgroup/dictionary/DeltaDictionary.java

```diff
@@ -20,26 +20,90 @@
 package org.apache.sysds.runtime.compress.colgroup.dictionary;
 
 import org.apache.commons.lang.NotImplementedException;
-import org.apache.sysds.runtime.functionobjects.Divide;
-import org.apache.sysds.runtime.functionobjects.Minus;
-import org.apache.sysds.runtime.functionobjects.Multiply;
-import org.apache.sysds.runtime.functionobjects.Plus;
+import org.apache.sysds.runtime.compress.colgroup.indexes.IColIndex;
+import org.apache.sysds.runtime.data.SparseBlock;
+import org.apache.sysds.runtime.functionobjects.*;
```

Comment:
we do not allow wildcard imports, instead import only the function objects in use,
There should be a setting for your IDE to change this.